README.md

```diff
@@ -25,6 +25,21 @@ Change your `Cargo.toml` to set the crate-type to `cdylib` (this instructs the c
 crate_type = ["cdylib"]
 ```
 
+### Rustup and wasm32-unknown-unknown installation
+
+Our example below will use the `wasm32-unknown-unknown` target. If this is not installed you will need to do so before this example will build. The easiest way to do this is use [`rustup`](https://rustup.rs/).
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
```

Comment:
The placement if this is a little strange because just a moment earlier, the user was running cargo, etc. So if they don't have rust installed they should be failing there right? Should we maybe move this up?

Reply:
yea I'd support that

Reply:
Good catch. I thought about that but assumed they probably had rust/cargo installed but maybe not rustup. Total noob so wasn't sure if rustup was part of the typical rust install.

Reply:
Actually.. should the comment and ### Rustup.. part be up at the top too? Thus this whole bit is at the top? Otherwise just the curl command may look a little odd.